xarray/tests/test_dataarray.py

```diff
@@ -4950,14 +4970,24 @@ def test_argmax(self, x, minindex, maxindex, nanindex):
 
         assert_identical(result3, expected2)
 
-    def test_idxmin(self, x, minindex, maxindex, nanindex):
-        ar0 = xr.DataArray(
+    @pytest.mark.parametrize("use_dask", [True, False])
+    def test_idxmin(self, x, minindex, maxindex, nanindex, use_dask):
+        if use_dask and not has_dask:
+            pytest.skip("requires dask")
+        if use_dask & (x.dtype.kind == "M"):
```

Comment:
it's normal to stick with `and` unless you need a vectorized or bitwise operation:
```suggestion
        if use_dask and x.dtype.kind == "M":
```